### DISCUSSION TITLE
APL pay

### DISCUSSION THREAD

**Discussion by Aisha1025** [woocommerce/woocommerce#48837] [Checkout Flow]
عند تنصيب بوابة الدفع  على المتجر لا يظهر الزر وعند مراجعه صفحة السداد يظهر لي 
![IMG_0919](https://github.com/woocommerce/woocommerce/assets/173693824/cbe63b30-430f-4ded-a543-0b7292acbf2a)

**Comment by nielslange** [comment]
Hello @Aisha1025 👋

Could you please describe the issue you're experiencing? The screenshot alone doesn't provide enough information to identify the problem. 

Thank you!
